src/NoteField.h

```diff
@@ -103,6 +104,9 @@ class NoteField : public ActorFrame
 	// This exists so that the board can be drawn underneath combo/judge. -Kyz
 	bool m_drawing_board_primitive;
 
+	// glow(for CalculatePulsingGlow)
+	mutable int m_iGlowCounter;
```

Comment:
Just make this a static variable in the function itself.

Reply:
Yeah i realized i should have done that after making the PR. I also made the int an int_fast8_t because it's perfectly fine if the value wraps and the range available doesn't matter.